static/js/tab_bar.js

```diff
@@ -322,6 +322,13 @@ function build_tab_bar() {
     reset_nav_bar();
 }
 
+$(document).on('zuliptypeaheadclosed', () => {
+    if ($("input:focus,textarea:focus")[0].className === "search-query input-block-level") {
```

Comment:
We should either add a comment here explaining what the function does, or just give the function a name (and then have a one-liner as needed to wire it up to the `$(document).on`, although ideally we can replace the document trigger mechanism with a more direct callback mechanism.)